libsql-server/src/rpc/replica_proxy.rs

```diff
@@ -42,29 +43,25 @@ impl ReplicaProxyService {
             .with(namespace.clone(), |ns| ns.jwt_key())
             .await;
 
-        let user_credential = parse_grpc_auth_header(req.metadata());
+            //todo julian figure this out
```

Comment:
comment formatting, and would probably be good to add some more information here on what there is to figure out.